implementations/Mavennet.json

```
@@ -10,11 +10,11 @@
   "issuers": [{
     "id": "did:key:z6MkiTsvjrrPNDZ1rrg9QDEYCFWCmEswT6U2cEkScb7edQ9b",
     "endpoint": "https://api.neo-flow.com/credentials/issueCredential",
-    "tags": ["VC-HTTP-API", "OAUTH2"]
+    "tags": ["OAUTH2"]
```

```suggestion
    "tags": ["OAuth2"]
```

We'll want to use proper capitalization in the tags.

---

All of the other tags are SHOUTY-KEBAB-CASE so this seems out of character.